sql-plugin/src/main/scala/com/nvidia/spark/rapids/Plugin.scala

```diff
@@ -408,6 +408,11 @@ class RapidsExecutorPlugin extends ExecutorPlugin with Logging {
       extraExecutorPlugins.foreach(_.init(pluginContext, extraConf))
       GpuSemaphore.initialize()
       FileCache.init(pluginContext)
+      // here customer need to make sure the path exists and is writable
+      val nsysStartComamnd = "nsys start -o /opt/spark/work-dir/test_%h_%p" +
```

Comment:
can we get the path from a ENV instead and make this path as a default one ?

Reply:
yeah that's doable and friendly to customer. They'll need to add code to do hdfs upload, so anyway they will touch the code.